pkg/k8sutil/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package k8sutil deals with talking to the Kubernetes API for verification
+// of the cluster and getting health of the cluster and nodes.
+// that need not be exposed outside.
```

Comment:
The documentation seems too specific for me.
```suggestion
// Package k8sutil simplifies interaction with Kubernetes API.
```